src/playercards/spawnbag/BondedBag.ttslua

```diff
@@ -0,0 +1,63 @@
+require("playercards/spawnbag/SpawnBag")
+
+SPAWN_SPEC = {
+  name = "BondedCards",
+  cards = {
+    "05314", -- Soothing Melody
+    "06277", -- Wish Eater
+    "06019", -- Bloodlust
+    "06022", -- Pendant of the Queen
+    "05317", -- Blood-rite
+    "06113", -- Essence of the Dream
+    "06028", -- Stars Are Right
+    "06025", -- Guardian of the Crystallizer
+    "06283", -- Unbound Beast
+    "06032", -- Zeal
+    "06031", -- Hope
+    "06033", -- Augur
+    "06331", -- Dream Parasite
+    "06015a", -- Dream-Gate
+  },
```

Comment:
Nit: list could be sorted by ID

Reply:
This maintains the ordering currently in the bag